### PR TITLE
Basic design

### PR DESCRIPTION
**Pull request by Thunae**
Just some content in the website, basic stuff with headers, divs and a form

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,28 @@
...
+          <a href="">Instagram </a>
+          <a href="">Email </a>
+      </div>
+  </header>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Ryan Gudal</h1>
       <p>This is my portfolio.</p>
+      <p>Hi my name is RYan Gudal, I'm a google intern.</p>
```
Comment (lyze) [inline portfolio/src/main/webapp/index.html]: s/RYan/Ryan/

Google

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -7,3 +7,13 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#header{
+    display: flex;
```
Comment (lyze) [inline portfolio/src/main/webapp/style.css:15]: nit: I think the indentation is a bit off. (2 vs 4 space)

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang= "en">
```
Comment (lyze) [inline portfolio/src/main/webapp/index.html:2]: I would suggest running the project through an HTML formatter to fix up the formatting inconsistencies.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Thunae) [reply portfolio/src/main/webapp/index.html]: Fixed

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (Thunae) [reply portfolio/src/main/webapp/style.css]: Done

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Thunae) [reply portfolio/src/main/webapp/index.html:2]: Done

**Pull request merged**